libraries/Microsoft.Bot.Builder.AI.QnA/Schemas/Microsoft.QnAMakerDialog.schema

```diff
@@ -110,6 +110,12 @@
             ]
         }
     },
+        "maximumScoreForLowScoreVariation": {
+           "type": "float",
+            "title": "maximumScoreForLowVariation",
+            "description": "Maximum score For Low Variation",
+            "default": 0.95F
+        },
```

Comment:
This needs to be in the properties section.  You are adding a property.

Reply:
Added as property